test/lib/mayaUsd/render/mayaToHydra/testPolygonPrimitives.py

```diff
@@ -26,6 +26,8 @@ class TestPolygonPrimitives(mtohUtils.MayaHydraBaseTestCase):
     IMAGE_DIFF_FAIL_THRESHOLD = 0.05
     IMAGE_DIFF_FAIL_PERCENT = 1.5
 
+    _requiredPlugins = ['modelingToolkit']
```

Comment:
This plugin was being loaded for all tests, in fixturesUtils.py, but only this test was using it.